### ISSUE TITLE
bmenu in manjaro i3 openrc 686 liveCD

### ISSUE THREAD

**Issue by Boruch-Baum** [Chrysostomus/bmenu#6]
Hi. Couldn't find an e-mail for you or the project, so here's an initial dump of my bmenu experience notes when evaluating the liveCD for manjaro i3 686. I really did not want to initially post any of this as a bug report before emailing you because I'm brand new to i3, and maybe some of what I'm experience may plausibly be an i3 or other issue; however, some clearly seem to be bmenu-specific

bmenu
- says it corrects the system time and timezone, but it doesn't
  update the time displayed on the panel
- selecting 1 for hardware, or 2 for display, surprisingly goes to the network to  synchronize the package database, and then asks me whether I want to install, and all that without telling me what it wants to install
- selecting 6 for network gets response "wrong option" in red,
  followed by "wait and try again later"
- selecting 4 for appearance, and then 5 for toggle dark theme, gets
  response "unknown option" in red, followed by "wait and try again
  later"
- for choosing the shell or the display manager, it would be helpful
  were there some indication of the current state.

Best to respond to me at: boruch_baum @@@at@@@ gmx ...com...

**Comment by Chrysostomus** [comment]
Hi!

Please open separate issues for different cases. 

1) time management is actually done by different package. You can open up issue there. Does tge time update on i3panel after you relogin?

2) this installs the necessary packages for the task ( mhwd-tui and brandr respectively). They are very small. Notification about it is a good idea. I thought pacman would give you the names, but appearantly it does not.

3) sounds like a bug. I'll look into it. Do you run openrc? Cause this option uses a systemd command to determine network manager first.

4&5) should not happen, sounds like a bug. I'll look into it.

6) good idea.

**Comment by Boruch-Baum** [comment]
That was quick! Thanks for the quick reply.

On 01/03/2016 08:24 PM, Chrysostomus wrote:

> Hi!
> 
> Please open separate issues for different cases. 
> Sure thing.
> 
> 1) time management is actually done by different package. You can open up issue there. Does tge time update on i3panel after you relogin?
> I'm still busy with the eval, but when I get to a point where I do
> logout and re login. I'll make note. Do you know offhand the name of
> that other package?

Wait. I just tried "restarting i3 in place" using mod+shift+r, and the
time is now displaying correctly.

Are you sure I should file this with the time management package? I have
no idea of the internals, but if your package is updating stuff, should
it be doing a config reload/restart, or does the environment depend upon
each package getting notified when a config file is modified?

> 2) this installs the necessary packages for the task ( mhwd-tui and brandr respectively). They are very small. Notification about it is a good idea. I thought pacman would give you the names, but appearantly it does not.
> My security consciousness gets me concerned that I'm not clearly warned
> in advance when a program attempts to go to the network. The current
> setup doesn't ask my permission before going out over the network to
> sync packages.
> 
> 3) sounds like a bug. I'll look into it. Do you run openrc? Cause this option uses a systemd command to determine network manager first.
> Yes, I'm running the openrc version. In fact, that's the reason I
> switched to manjaro from debian. When debian adopted systemd, I was one
> of those who bailed. I've been using manjaro-openrc with xfce for the
> past few months, and have been reasonably happy; when I saw the i3
> release announcement, I thought I would check out your spin.
> 
> 4&5) should not happen, sounds like a bug. I'll look into it.
> 
> 6) good idea.
> Glad to be of some help.

So, in summary: I'll file 2-5 as separate bugs, 6 as an enhancement
request, and for 1, you'll update me if you know the other package name,
or if you want to do something like have bmenu "restart i3 in place".

> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Chrysostomus/bmenu/issues/6#issuecomment-168562708

## 

hkp://keys.gnupg.net
CA45 09B5 5351 7C11 A9D1  7286 0036 9E45 1595 8BC0

**Comment by Chrysostomus** [comment]
If it works after reloadong i3, it is feature, not a bug. It is a small bash script and cannot support live reloadong on every enviroment. Name of the package is timeset.

**Comment by Boruch-Baum** [comment]
Thanks. I'll drop the issue, then.

On 01/03/2016 09:32 PM, Chrysostomus wrote:

> If it works after reloadong i3, it is feature, not a bug. It is a small bash script and cannot support live reloadong on every enviroment. Name of the package is timeset. 
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Chrysostomus/bmenu/issues/6#issuecomment-168567741

## 

hkp://keys.gnupg.net
CA45 09B5 5351 7C11 A9D1  7286 0036 9E45 1595 8BC0
